js/models/vehicle.ts

```diff
@@ -132,3 +132,14 @@ export const lateForNext = (vehicle: Vehicle): number | null => {
 
   return minutesAfter(estimated, nextDeparture);
 };
+
+export const latestOcsUpdatedAt = (vehicle: Vehicle): DateTime | null => {
+  const datetimes = [vehicle.ocsTrips.current, vehicle.ocsTrips.next[0]]
```

Comment:
question: necessary to check the length of `next`?

Reply:
I personally think it's OK as-is, but I can add an explicit check if you prefer? If `next[0]` is `undefined`, it gets stripped out in the filter call. (Also note that there is a unit test case that exercises that situation.)

Reply:
oh! 🤦 it's fine as is, then